src/components/input/radio/index.tsx

```diff
@@ -0,0 +1,76 @@
+import {
+	Component,
+	ComponentWillLoad,
+	Event,
+	EventEmitter,
+	h,
+	Host,
+	Listen,
+	Method,
+	Prop,
+	VNode,
+	Watch,
+} from "@stencil/core"
+import { Color, Data } from "../../../model"
+import { Clearable } from "../Clearable"
+import { Input } from "../Input"
+import { Looks } from "../Looks"
+import { Selectable } from "./Selected"
+
+@Component({
+	tag: "smoothly-input-radio",
+	styleUrl: "style.css",
+	scoped: true,
+})
+export class SmoothlyInputRadio implements Input, Clearable, ComponentWillLoad {
+	active?: Selectable
+	@Prop({ mutable: true }) value: any = undefined
+	@Prop({ mutable: true, reflect: true }) looks: Looks = "plain"
+	@Prop() clearable?: boolean
+	@Prop() name: string
+	@Event() smoothlyInputLooks: EventEmitter<(looks: Looks, color: Color) => void>
+	@Event() smoothlyInput: EventEmitter<Data>
+	@Event() smoothlyInputLoad: EventEmitter<(parent: HTMLElement) => void>
+	componentWillLoad(): void | Promise<void> {
+		this.smoothlyInputLooks.emit(looks => (this.looks = looks))
+		this.smoothlyInput.emit({ [this.name]: this.value })
```

Comment:
This input have not yet received information about if one of its children have a value.
![image](https://github.com/utily/smoothly/assets/25643315/87bfbda7-da46-41ca-960d-71d61fcb4b74)
Maybe move this emit into `componentDidLoad()`.

Reply:
also add demo where one of the options are initially set to make sure that it works as intended. i guess it should emit undefined if none of the options are selected initially.

Reply:
moving it to componentDidLoad() did not work, i suggest that the button (the child) emits value (if it has one) in its componentWillLoad cause that works! Sounds alright?

Reply:
> i suggest that the button (the child) emits value (if it has one) in its componentWillLoad cause that works! Sounds alright?

Yes this should be done. But you could also check if you have received an event from a child before `componentDidload()` is run. We know that we can check for that since since [all the `componentWillLoad()`s are run prior to `componentDidLoad()`](https://stenciljs.com/docs/component-lifecycle#lifecycle-hierarchy). 

1. So I would suggest making a private variable (no state / prop) that is a boolean defaulted to `false` that is set to `true` if it receives an event from one of the children.
2. Inside `componentDidLoad()` if the variable is still set to `false` allow it to emit. Otherwise assume it already emitted a value from earlier.

This way you should get:
* only 1 emit no matter what
* emit value on initial load
* emit undefined if no button is defaulted to selected

Also please add another set of radio buttons where you allow one of the radio buttons to be selected so we can see that both of them behaves as expected.